packages/playwright-core/src/server/injected/highlight.ts

```diff
@@ -113,6 +113,27 @@ export class Highlight {
     this._actionPointElement.hidden = true;
   }
 
+  blackoutElements(elements: Element[]) {
+    const boxes = elements.map(e => e.getBoundingClientRect());
+    const pool = this._highlightElements;
+    this._highlightElements = [];
+    for (const box of boxes) {
+      const highlightElement = pool.length ? pool.shift()! : this._createHighlightElement();
+      highlightElement.style.backgroundColor = '#000';
```

Comment:
As dark and light themes are common place, we can't hardcode a `#000` overlay color.

Instead we should add an `overlayColor` ( or whatever more inclusive name we pick for this feature ) property that defaults to a neutral or less common color e.g. `#808080` or `#F0F`

Reply:
I like the `#F0F`!